app/views/catalog/_search_results.html.erb

```diff
@@ -7,6 +7,10 @@
   <%= json_api_link_tag %>
 <% end %>
 
+<% content_for(:skip_links) do -%>
+  <a href="#documents" class="element-invisible element-focusable" data-turbolinks="false">Skip to first result</a>
```

Comment:
I believe we could use `link_to` here (although not super important).  Also, we should internationalize this value. 

I'm not 100% sure what our process is for adding new UI strings these days (e.g. skip the specs that will fail if we add a new key to only one language, use Google Translate to translate the value for us, some other option).

Reply:
I've been using Google Translate 🤷‍♂